components/centraldashboard/webpack.config.js

```diff
@@ -48,10 +48,15 @@ const COMPONENT_RULES = [
 
 module.exports = {
     mode: ENV,
-    entry: resolve(SRC, 'index.js'),
+    entry: {
+        app: resolve(SRC, 'index.js'),
+        dashboard_lib: resolve(SRC, 'library.js'),
+    },
     output: {
         filename: '[name].bundle.js',
```

Comment:
```suggestion
         filename: 'centraldashboard.sdk.js',
```

Reply:
That will not work because the [name] placeholder gets replaced by the item from the `entry` property (ie. app for the application code, dashboard_lib for the library). That's why I renamed the entry to dashboard_lib so that the emitted output is `dashboard_lib.bundle.js`. `centraldashboard_lib.bundle.js` felt too long.